### DISCUSSION TITLE
composer create-project fails

### DISCUSSION THREAD

**Discussion by Wollitoo** [pimcore/pimcore#17687] [Q&A / Support] [answered]
I am trying to initialize a fresh Pimcore 2024.3 project but the composer create-project fails. I doesn't matter if I use ddev composer create or composer create-project directly. I tried it on multiple system with linux and macOS. I followed the steps listed on the skeletons github page (https://github.com/pimcore/skeleton) and the steps listed on the ddev page (https://ddev.readthedocs.io/en/stable/users/quickstart/#pimcore). I've been using Pimcore as a dev for 3 years now and never encountered this behavior before.

I also tried installing it via 'composer require pimcore/skeleton' and calling 'composer install' directly inside of a container which sort of worked. The issue for that instance is that I can't create data objects in the backend UI because the implementation loader for my class isn't generated. It also look like that the composer package "pimcore/platform-version" is missing. If I try to get it via "composer require pimcore/platform-version" I get the same error as before.

Composer version: 2.8.1 | ddev version: v1.23.4

Error: 
![image](https://github.com/user-attachments/assets/525c88cf-f4b4-4c19-b557-81b6e855e9d2)

**Comment by fashxp** [comment]
just did a ```docker run -u `id -u`:`id -g` --rm -v `pwd`:/var/www/html pimcore/pimcore:php8.3-latest composer create-project pimcore/skeleton my-project``` and it ran without problems...

**Reply by Wollitoo** [reply, marked as answer]
Yeah the docker thingy worked. To make it work locally and in ddev we had to change the symfony/runtime version in the composer.json from ^6.2 to ^7.1 and remove the dependencies to "symfony/amqp-messenger": "^6.2" and "symfony/dotenv": "^6.2". Pimcore seems to be working just fine now.

**Reply by fashxp** [reply]
strange thing, thx for sharing.

**Reply by Wollitoo** [reply]
I was able to add "symfony/dotenv":"^7.1" (bumped this version too) back in after the successful Pimcore installation. It is very strange. I will close this for now :) thx for the help

**Reply by blankse** [reply]
@Wollitoo It seems to be a bug in Composer 2.8.*
See https://github.com/pimcore/skeleton/issues/202
